src/main/proto/ga4gh/assay_metadata.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "google/protobuf/struct.proto";
+
+
+// An experimental preparation of a sample.
+message Experiment {
+  // The experiment UUID. This is globally unique.
+  string id = 1;
+
+  // The name of the experiment.
+  string name = 2;
+
+  // A description of the experiment.
+  string description = 3;
+
+  // The time at which this message was created.
+  // Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
+  string message_create_time = 4;
```

Comment:
It's in the master `metadata.proto`
https://github.com/ga4gh/schemas/blob/5ccf897fc1ee9c0401db0b2d117e80d87388bec9/src/main/proto/ga4gh/metadata.proto#L32 . Stubbed out this issue here: https://github.com/ga4gh/schemas/issues/611